frame/support/procedural/src/pallet/expand/call.rs

```diff
@@ -31,7 +32,7 @@ pub fn expand_call(def: &mut Def) -> proc_macro2::TokenStream {
 
 			(span, where_clause, methods, docs)
 		}
-		None => (def.pallet_struct.attr_span, None, Vec::new(), Vec::new()),
+		None => (def.item.span(), None, Vec::new(), Vec::new()),
 	};
```

Comment:
I'm not sure it makes sense anymore to declare a dummy Call enum in case `#[pallet::call]` is not define, because construct_runtime will forbid its usage.
So the dummy Call enum should not be usable anymore. Maybe we can do in a follow-up though